src/io/utils.rs

```diff
@@ -47,45 +46,19 @@ use crate::io::{
 };
 use crate::logger::{log_error, LdkLogger, Logger};
 use crate::peer_store::PeerStore;
-use crate::types::{Broadcaster, DynStore, KeysManager, Sweeper, WordCount};
+use crate::types::{Broadcaster, DynStore, KeysManager, Sweeper};
 use crate::wallet::ser::{ChangeSetDeserWrapper, ChangeSetSerWrapper};
 use crate::{Error, EventQueue, NodeMetrics, PaymentDetails};
 
 pub const EXTERNAL_PATHFINDING_SCORES_CACHE_KEY: &str = "external_pathfinding_scores_cache";
 
-/// Generates a random [BIP 39] mnemonic with the specified word count.
-///
-/// If no word count is specified, defaults to 24 words (256-bit entropy).
-///
-/// The result may be used to initialize the [`Node`] entropy, i.e., can be given to
-/// [`Builder::set_entropy_bip39_mnemonic`].
-///
-/// [BIP 39]: https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki
-/// [`Node`]: crate::Node
-/// [`Builder::set_entropy_bip39_mnemonic`]: crate::Builder::set_entropy_bip39_mnemonic
-pub fn generate_entropy_mnemonic(word_count: Option<WordCount>) -> Mnemonic {
-	let word_count = word_count.unwrap_or(WordCount::Words24).word_count();
-	Mnemonic::generate(word_count).expect("Failed to generate mnemonic")
-}
-
-pub(crate) fn read_or_generate_seed_file<L: Deref>(
-	keys_seed_path: &str, logger: L,
-) -> std::io::Result<[u8; WALLET_KEYS_SEED_LEN]>
-where
-	L::Target: LdkLogger,
-{
+pub(crate) fn read_or_generate_seed_file(
```

Comment:
would be good to have a test for this function, that it can read the seeds it writes

Reply:
Done.